.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,61 @@
+name: Floris Benchmark
+on:
+  schedule:
+    - cron: '0 3 * * *'  # Runs daily at 3am UTC
```

Comment:
I suggest tying this to a commit rather than a date since any given day might contain multiple commits and many will contain no commits.

Reply:
Yeah, that makese sense, maybe commits to develop?